Dockerfile

```diff
@@ -95,14 +95,29 @@ RUN cd /tmp && \
 # Install PostgreSQL in a dedicated conda environment.
 RUN conda create -c conda-forge -n pgsql postgresql=10 && conda clean --all -f -y
 
+# Install RabbitMQ in a dedicated conda environment.
+# If the architecture is arm64, we install the default version of rabbitmq provided by Ubuntu.
+RUN if [ "$TARGETARCH" = "amd64" ]; then \
+   conda create -c conda-forge -n rmq rabbitmq-server=3.8.14 && conda clean --all -f -y; \
+elif [ "$TARGETARCH" = "arm64" ]; then \
+   apt-get update && apt-get install -y --no-install-recommends  \
+       rabbitmq-server && \
+       rm -rf /var/lib/apt/lists/* && \
+       apt-get clean all; \
+else \
+   echo "Unknown architecture: ${TARGETARCH}."; \
+fi
+
+
 # Copy the script load-singlesshagent.sh to /usr/local/bin.
 COPY bin/load-singlesshagent.sh /usr/local/bin/load-singlesshagent.sh
 
 # Create system user.
 COPY my_init.d/create-system-user.sh /etc/my_init.d/10_create-system-user.sh
 
 # Launch rabbitmq server
-COPY my_init.d/start-rabbitmq-${TARGETARCH}.sh /etc/my_init.d/20_start-rabbitmq.sh
+COPY opt/start-rabbitmq-${TARGETARCH}.sh /opt/start-rabbitmq.sh
+COPY my_init.d/start-rabbitmq.sh /etc/my_init.d/20_start-rabbitmq.sh
```

Comment:
I am confused with these. 

Ah, I see. The `/opt/start-rabbitmq.sh` will be different for arm64 and amd64, and start-rabbitmq.sh will used for both?

Reply:
> Ah, I see. The `/opt/start-rabbitmq.sh` will be different for arm64 and amd64, and start-rabbitmq.sh will used for both?

yep!